DemoApp/DemoApp/Helpers/ServiceProviderUtils.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+namespace DemoApp.Helpers
+{
+    public class ServiceProviderUtils
+    {
+        public static TService GetService<TService>() =>
+                    Current.GetService<TService>();
+
+        public static IServiceProvider Current =>
+#if ANDROID
+    MauiApplication.Current.Services;
+#elif IOS || MACCATALYST
+			MauiUIApplicationDelegate.Current.Services;
+#else
+            _current;
```

Comment:
_current will be undefined when you run in Windows. Can you check in Windows once?